doc/changes/latest.inc

```diff
@@ -124,6 +124,8 @@ Bug
 
 - Fix :class:`mne.Report` silently suppressing exceptions when used as a context manager by `Marijn van Vliet`_
 
+- Fix bug in :func:`mne.io.reference._apply_reference` to set `info.custom_ref_applied` to `True` for 'ecog' and 'seeg' channels in addition to 'eeg' by `Liberty Hamilton`_
```

Comment:
FYI, "True" needs *two* backticks on either side of it to get properly formatted, not one (it's reStructuredText, not Markdown). This won't error out, it will just make it italicized instead of formatted as code.